--- a/Makefile	2026-01-28 16:24:22.927638764 +0100
+++ b/Makefile	2026-01-28 23:12:01.191139222 +0100
@@ -6,7 +6,7 @@
 
 #  configuration parameters
 name=unspecified
-dest=/must/specify/dest/
+dest="${DESTDIR}/usr"
 
 
 ##################################################################
@@ -89,16 +89,15 @@
 
 D=$(dest)
 Install:
-		mkdir $D
-		mkdir $D/bin $D/lib $D/doc $D/man $D/man/man1
+		mkdir -p $D/bin $D/share/doc/icon $D/share/man/man1
-		cp README $D
+		cp README $D/share/doc/icon
 		cp bin/[cflpvwx]* $D/bin
 		cp bin/icon[tx]* $D/bin
 		rm -f $D/bin/libI*
 		(cd $D/bin; ln -s icont icon)
 		cp lib/*.* $D/lib
-		cp doc/*.* $D/doc
+		cp doc/*.* $D/share/doc/icon
-		cp man/man1/*.* $D/man/man1
+		cp man/man1/*.* $D/share/man/man1
 
 
 # Bundle up for binary distribution.
